Automattic/newspack-plugin
# feat: merge `trunk` into `epic/ia` + fix conflicts

### All Submissions:


### Changes proposed in this Pull Request:

Merge `trunk` into `epic/ia` + necessary fixes to comply with new package constraints introduced in WP 6.6 upgrade.

### How to test the changes in this Pull Request:

1. Checkout branch and build assets - `git checkout fix/epic-ia-trunk && npm run build`.
2. Navigate to _/wp-admin/admin.php?page=newspack-dashboard#/_, confirm everything works as expected (Data loads correctly, no console errors etc).
3. Navigate to _/wp-admin/admin.php?page=newspack-settings#/_, confirm everything works as expected (Tabs transition, data saves, no errors etc).

### Other information:
